docs/api.md

```diff
@@ -462,6 +463,11 @@ Shortcut for [page.mainFrame().focus(selector)](#framefocusselector).
 #### page.frames()
 - returns: <[Array]<[Frame]>> An array of all frames attached to the page.
 
+#### page.getContent()
```

Comment:
let's go with `page.content()`, we don't have "get" prefixes in other API calls (e.g. [`page.plainText()`](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pageplaintext)).